tests/sql/glam-fenix-dev/glam_etl/org_mozilla_fenix_glam_nightly__clients_histogram_aggregates_v1/test_minimal/data.py

```diff
@@ -0,0 +1,97 @@
+"""Testing data for query."""
+
+from pathlib import Path
+
+import yaml
+
+ROOT = Path(__file__).parent
+
+UUID = "df735f02-efe5-4b07-b212-583bb99ba241"
+SUBMISSION_DATE = "2020-10-01"
+APP_BUILD_ID = "2020100100"
+
+LATEST_VERSION = [dict(channel="nightly", latest_version=84)]
+
+# NOTE: what happens when channel = "*"?
+CLIENTS_HISTOGRAM_AGGREGATES = [
+    {
+        "sample_id": 1,
+        "client_id": UUID,
+        "ping_type": "metrics",
+        "os": "Android",
+        "app_version": 84,
+        "app_build_id": APP_BUILD_ID,
+        "channel": "nightly",
+        "histogram_aggregates": [],
```

Comment:
Although not exactly minimal, it would be more interesting if the test summed onto an existing aggregate

Reply:
Added a separate test summing onto an existing aggregate.